wpical/src/main/native/cpp/fmap.cpp

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+#include "fmap.h"
+
+#include <string>
+
+std::string fmap::singleTag(int tag, tag::pose tagpose, bool endTag) {
```

Comment:
```suggestion
std::string fmap::singleTag(int tag, const tag::pose& tagpose, bool endTag) {
```

Reply:
Also. Is the all lowercase pose struct name in line with our coding standards?

Reply:
No, coding guidelines are for all classes and structs to use PascalCase.